R/favorites.R

```diff
@@ -1,22 +1,21 @@
-#' Get tweets data for statuses favorited by one or more target users.
+#' Get tweets favorited by one or more users
 #'
-#' Returns up to 3,000 statuses favorited by each of one or more
-#' specific Twitter users.
+#' Returns up to 3,000 tweets favorited for each `user`.
 #'
 #' @inheritParams TWIT_paginate_max_id
 #' @inheritParams get_timeline
 #' @return A tibble with one row for each tweet.
 #' @examples
 #' \dontrun{
-#'
-#' ## get max number of statuses favorited by KFC
-#' kfc <- get_favorites("KFC", n = 3000)
+#' # get likes for a single user
+#' kfc <- get_favorites("KFC", n = 600)
 #' kfc
+#' # get newer likes since last request
+#' newer <- get_favorites("KFC", since_id = kfc)
```

Comment:
Perhaps restrict kfc to some rows to show some results otherwise unless they tweet between these two calls newer will be always empty (this also happens on other examples)

Reply:
Ok, I'll remove it from the example here and instead put it in `search_tweets` where it is likely you'll see new data.